src/api/app/policies/comment_policy.rb

```diff
@@ -27,4 +27,10 @@ def update?
 
     user == record.user
   end
+
+  def reply?
+    return false if user.blank? || user.is_nobody? || record.user.is_nobody?
+
+    true
```

Comment:
```suggestion
    !(user.blank? || user.is_nobody? || record.user.is_nobody?)
```
`return false if ...` and in the next sentence return `true` looks redundant to me. What do you think?